pkg/machines/scripts/create_machine.sh

```diff
@@ -32,27 +31,20 @@ trap err_handler EXIT
 
 XMLS_FILE="`mktemp`"
 ROOT_PASSWORD_FILE="`mktemp`"
-USER_PASSWORD_FILE="`mktemp`"
 
 if [ "$UNATTENDED" = "true" ]; then
     if [ -z "$ROOT_PASSWORD" ]; then
         ROOT_PASSWORD_PARAM=''
     else
         echo "$ROOT_PASSWORD" > "$ROOT_PASSWORD_FILE"
-        ROOT_PASSWORD_PARAM=",admin-password-file=$ROOT_PASSWORD_FILE"
-    fi
-    if [ -z "$USER_PASSWORD" ]; then
-        USER_PASSWORD_PARAM=''
-    else
-        echo "$USER_PASSWORD" > "$USER_PASSWORD_FILE"
-        USER_PASSWORD_PARAM=",user-password-file=$USER_PASSWORD_FILE"
+        ROOT_PASSWORD_PARAM="admin-password-file=$ROOT_PASSWORD_FILE"
```

Comment:
so before it created `--unattended ,admin-password-file=foo`, now the coma has been dropped, right? Looking into man pages it does not say how sub options should be separated, but I guess comma separated. So it would just work with leading comma but it not actually needed?

Reply:
Exactly. The main fix is above in the rm -f where all file names where wrapped in double quotes, which obviously did not work.